addons/mail_bot/__manifest__.py

```diff
@@ -8,18 +8,14 @@
     'summary': 'Add OdooBot in discussions',
     'description': "",
     'website': 'https://www.odoo.com/page/discuss',
-    'depends': ['mail'],
+    'depends': ['mail', 'mail_bot_hr'],
```

Comment:
Actually that's the inverse. mail_bot_hr should be the bridge between mail and hr . Mail_bot is the base module, mail_bot_hr adds some features in it.